websocket_back/src/room.js

```diff
@@ -0,0 +1,80 @@
+export default class Room {
+
+  constructor (options) {
+    this.io = options.io;
+    this.socket = options.socket;
+    this.roomId = options.roomId;
+    this.username = options.username;
+    // join | create
+    this.action = options.action;
+    this.store = this.io.adapter;
+  }
+
+  async init () {
+
+    let clients;
+    await this.io.in(this.roomId).clients((e, _clients) => {
+      clients = _clients || console.error('[INTERNAL ERROR] Room creation failed!');
+      console.log(`Connected Clients are: ${clients}`);
+    });
+
+    if (this.action === 'join') {
+
+      if (clients.length >= 1) {
+        await this.socket.join(this.roomId);
+        this.store.clients.push({ id: this.socket.id, username: this.username, isReady: false });
+        this.socket.username = this.username;
+        this.socket.emit('join_room', '[SUCCESS] Successfully initialised');
```

Comment:
Add the room id to the response
```suggestion
        this.socket.emit('join_room', '[SUCCESS] Successfully initialised', this.roomId);
```